### ISSUE TITLE
"Filter pandoc-fignos not found"

### ISSUE THREAD

**Issue by julianaus** [tomduck/pandoc-fignos#33]
Apologies for my very basic question, but I am not a coder and I don't get pandoc-fignos to work. Installing it worked fine. But the filter still does not seem to be there. After trying to use the filter employing this command in Terminal

    pandoc --filter pandoc-fignos TableFigure.md -f markdown -t html -s -o TableFigure.html

I always get the error message:
> pandoc: Filter pandoc-fignos not found

Any idea how I can solve this? 
Best,
Julian

**Comment by tomduck** [comment]
Hi @julianaus.  I'm sure that we can sort this out.  Can you please tell me what kind of system you are working on (Mac/Linux/Windows/etc) and how you installed the filter?  Thanks, Tom.

**Comment by julianaus** [comment]
Hi @tomduck, thank you. I have a Mac and used the first installing option you proposed:

    pip install pandoc-fignos 

Installing did not work immediately - I had to install Pip and change some folder rights, but in the end it worked. If run the install comand now, I get the following message:

> ausse$ pip install pandoc-fignos 
> Requirement already satisfied: pandoc-fignos in /Library/Python/2.7/site-packages
> Requirement already satisfied: pandoc-xnos>=0.7 in /Library/Python/2.7/site-packages (from pandoc-fignos)
> Requirement already satisfied: pandocfilters in /Library/Python/2.7/site-packages (from pandoc-fignos)
Y Requirement already satisfied: pandoc-attributes in /Library/Python/2.7/site-packages (from pandoc-fignos)
Y Requirement already satisfied: psutil>=4.1.0 in /Library/Python/2.7/site-packages (from pandoc-xnos>=0.7->pandoc-fignos)

I assume I can still run pandoc in Terminal, right? Or do I have to use it in Python if I want to use pandoc-fignos?

**Comment by tomduck** [comment]
Hi @julianaus.  I think I see the problem.  You need to install pandoc-fignos as root.  On a Mac that means using:

~~~
$ sudo pip install pandoc-fignos
~~~

To upgrade, use

~~~
$ sudo pip install --upgrade pandoc-fignos
~~~

You will need to enter your administrator password when asked.  I will make the documentation more clear on this point.

Once pandoc-fignos is installed, you can test for it in Terminal as follows:

~~~
$ which pandoc-fignos
/Library/Frameworks/Python.framework/Versions/3.4/bin/pandoc-fignos
~~~

That's for my Mac; you will see something different.  If `which` gives nothing, then there is a problem.

Please let me know if this works for you.

In answer to your other question: Yes, you can still run pandoc in Terminal.  Once pandoc-fignos is installed, the language becomes irrelevant.  It should just work.

**Comment by julianaus** [comment]
Amazing, @tomduck. It worked. Thank you very much for your help.

**Comment by julianaus** [comment]
Best regards from Vienna! Julian

**Comment by tomduck** [comment]
Fantastic!  I added some text to make the install process a little clearer.  Thanks very much for your feedback.

... and greetings from Halifax (Canada). :o)

Cheers,
Tom

**Comment by AlejoOdgers** [comment]
Hello! Sorry for my basic question too... I have the same problem, with a very similar message (just different versions) when i try to run the pip instal command again. The only difference is that I am in Windows. I looked for alternatives to the "$ sudo" command in windows, but what I tried didn't work, and I couldn't understand it very well... Sorry for asking something so basic. 

Have a great day!

Alejo.
